packages/labs/src/common/index.ts

```diff
@@ -4,6 +4,5 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
-import * as classes from "../common/classes";
-
-export const Classes = classes;
+// Empty module since there's nothing to export.
+export {};
```

Comment:
...I guess so, then.

Reply:
I thought we were planning to re-export `@blueprintjs/select` components? That can be another PR though.

Reply:
future PR, I still havea a few reservations